static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -130,6 +131,14 @@
              class="admin-realm-message-content-edit-limit-minutes"
              value="{{ realm_waiting_period_threshold }}"/>
     </div>
+    <div class="input-group">
+      <label for="realm_notifications_stream">{{t "Notifications stream" }}:</label>
+      <select name="realm_notifications_stream" id="id_realm_notifications_stream">
+        {{#each stream_list}}
+        <option value='{{this.name}}'>{{this.name}}</option>
+        {{/each}}
+      </select>
+    </div>
```

Comment:
UI-wise, I think we should probably have a "Change" link that opens a modal here, rather than having the selection menu in the page, since a dropdown menu is kinda cluttered